README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
- [ ] The project requirement suggests that we should install Redux Toolkit as a dependency. Please install the required packages and make sure that they are inside the `package.json` file. 😎

```
Project requirements
 - Add Redux Toolkit (npm install react-redux @reduxjs/toolkit).
```